charts/simple-app/values.yaml

```diff
@@ -342,7 +342,7 @@ autoscaling:
   # -- Configures the HPA to target a particular CPU utilization percentage
   targetCPUUtilizationPercentage: 80
   # -- If supplied, configures the HPA to target a particular Memory utilization percentage
-  # targetMemoryUtilizationPercentage: 80
```

Comment:
Eh. I think it should be uncommented, but also left blank. You don't generally want to autoscale on CPU and Memory in _most_ cases without lots of tuning. That's why it was left off here. It doesn't change the capability of the downstream chart from setting the value though.

Reply:
ooh yea that makes sense, i'll leave it as is